posts/how-not-communicate/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: How not to communicate with your team
+date: 2020-02-06
+description: The story of three communication mistakes I made with my team, and how we resolved them. This is the script of the last talk I gave at Algolia, before leaving.
+---
+
+<style>
+  img {
+    display: block;
+    width: 600px;
+    margin: 32px auto;
+  }
+</style>
+
+# How *Not* to Communicate With Your Team
+
+Most of my career (so far), I have lead teams as a "[benevolent dictator](https://producingoss.com/en/benevolent-dictator.html)". Meaning that I have the vision of what we want to achieve as a team, but I am also the one prescribing the way we implement that vision. As a consequence, I expect my team to follow my lead, and to object just if I overlook anything important. Implementing this leadership style has been very effective, until I joined Algolia.
+
+Besides the fact that, at Algolia, my team and I had more years of experience than my previous teams, the culture of the company made it ineffective to keep implementing this leadership style. So, with help from my manager, I had to take a step back and develop my soft skills in order to lead effectively in that new organization.
```

Comment:
You can link my Linkedin account on "my manager" if you want to 😛 https://www.linkedin.com/in/daguenet/